tests/kernel_bundle/kernel_bundle_spec_const.cpp

```diff
@@ -0,0 +1,104 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Copyright (c) 2023 The Khronos Group Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+*******************************************************************************/
+
+#include "kernel_bundle.h"
+
+namespace kernel_bundle_spec_const {
+
+constexpr sycl::specialization_id<int> SpecName(5);
+constexpr sycl::specialization_id<int> OtherSpecName(5);
```

Comment:
I presume the 5 is just a randomly chosen number, but the fact that two different ids use the same number is intentional, because the ids should still be the same? I think that should probably have a comment about that.

Reply:
No, they don't have to be the same, changed it to different value to avoid confusion.